gen/quick_test.go

```diff
@@ -260,6 +266,30 @@ func TestQuickRoundTrip(t *testing.T) {
 					}, "failed to String %#v", give)
 				}
 			})
+
+			if !tt.NoLog {
+				t.Run("Zap", func(t *testing.T) {
+					for _, give := range values {
+						assert.NotPanics(t, func() {
```

Comment:
Is this necessary? What can panic here?

Reply:
It panics if we see an unknown type instead of having some default behavior. It shouldn't actually ever panic because we've covered every type.

Reply:
Well, nothing should panic, but in that case I also wanted a function I could
`return` from for flow control.

Reply:
sgtm